extensions/amp-image-viewer/0.1/amp-image-viewer.js

```diff
@@ -465,6 +465,7 @@ export class AmpImageViewer extends AMP.BaseElement {
     // Movable.
     this.unlistenOnSwipePan_ = this.gestures_
         .onGesture(SwipeXYRecognizer, e => {
+          event.preventDefault();
```

Comment:
looks like all the gesture handles preventDefault now. Maybe `Gestures.get(` doesn't need to the `/* opt_shouldNotPreventDefault */true` anymore? then we can remove all the `event.preventDefault();`.

Only one to double check is `onPointerDown` which preventsDefaults conditionally. Not sure what happens if it does it in all cases.

Approving the PR and leavings this to your discretion.

Reply:
The reason why it has to be like this is because we only want to preventDefault AFTER the gesture triggers, but not when an event triggers a pending gesture. `/* opt_shouldNotPreventDefault */false` results in preventing default being called on single taps, which is why it broke lightbox toggle. 

This should have happened in https://github.com/ampproject/amphtml/pull/12930.